ide/xmlprotocol.mli

```diff
@@ -29,6 +29,8 @@ val set_options : set_options_sty -> set_options_rty call
 val quit        : quit_sty        -> quit_rty call
 val init        : init_sty        -> init_rty call
 val stop_worker : stop_worker_sty -> stop_worker_rty call
+(* testing *)
```

Comment:
Please be more explicit, something such as `private to fake_ide. Not for public use`

Reply:
Ok